python_ta/cfg/visitor.py

```diff
@@ -86,6 +86,8 @@ def visit_if(self, node: nodes.If) -> None:
 
         # Handle "then" branch.
         then_block = self._current_cfg.create_block(old_curr)
+        # Label the edge to the "then" branch
+        self._current_cfg.label_edge(old_curr, then_block, True)
```

Comment:
So overall this approach is pretty good, but has the downside that it requires a bit of iteration through existing edges. This isn't necessary in all the cases you've added here, which explicitly match a specific source-target pair.

So a better approach is to *pass the label as an optional argument to `create_block`*, and from there pass it down to `link_or_merge`, and to the Edge initializer. This would make the above code (on line 88) look like

```python
then_block = self._current_cfg.create_block(old_curr, edge_label='True')
```

or something. I would also pass in `'True'` rather than `True` to avoid confusion (but it's fine to leave the label as `Any`, I could see storing more information there in the future perhaps).